### PR TITLE
arm64/L4T Jetson support

### PR DESCRIPTION
**Pull request by ds58**
- Build Linux binaries on x86_64 and arm64 in stereolabs/zed docker containers
- Support for arm64/L4T

### CODE REVIEW

**Comment by ds58** [comment]
Workflow run for building natives https://github.com/ihmcrobotics/zed-java-api/actions/runs/10964017718

File: src/main/java/us/ihmc/zed/SL_IMUData.java [hunk 1]
```diff
@@ -49,7 +49,7 @@ public class SL_IMUData extends Pointer {
 	*/
 	public native @ByRef SL_Vector3 angular_velocity(); public native SL_IMUData angular_velocity(SL_Vector3 setter);
 	/**
-	\brief Linear acceleration vector (3x1) of the gyroscope in m/s².
```
Comment (TomaszTB) [inline src/main/java/us/ihmc/zed/SL_IMUData.java:52]: Huh, it lost the special characters

**Review by TomaszTB — APPROVED** [review]
Awesome! How long does the native build workflow usually run?

File: src/main/java/us/ihmc/zed/SL_IMUData.java (hunk 1 above)
Comment (ds58) [reply src/main/java/us/ihmc/zed/SL_IMUData.java:52]: Yeah I don't really know why it happened and I guess it's not so important

**Comment by ds58** [comment]
> Awesome! How long does the native build workflow usually run?

I think around an hour

**Pull request merged**